packages/react/dialog/package.json

```
@@ -41,7 +41,7 @@
     "@radix-ui/react-slot": "workspace:*",
     "@radix-ui/react-use-controllable-state": "workspace:*",
     "aria-hidden": "^1.1.1",
-    "react-remove-scroll": "2.6.0"
+    "react-remove-scroll": "2.6.1"
```

Can we go ahead and switch this to allow for a range? This should prevent the need for continually updating Radix every time a new patch or minor is shipped. Anton's pretty solid about following semver.
```suggestion
    "react-remove-scroll": "^2.6.1"
```